data/category-porn

```diff
@@ -2206,6 +2214,7 @@ ichineseporn.com
 icuminside.com
 ideal-teens.com
 idol-sagasukun.com
+iframejav.com
```

Comment:
Although the website blocked my location and all the proxy I have, I think that is a pornographic website.